docs/reference/edot-php/configuration.md

```
@@ -18,7 +18,7 @@ Learn how to configure the {{edot}} PHP (EDOT PHP) to send data to Elastic.
 Because the {{edot}} PHP is an extension of the OpenTelemetry PHP SDK, it supports:
 
 * [OpenTelemetry configuration options](#opentelemetry-configuration-options)
-* [Configuration options only available in EDOT PHP](#options-only-available-in-edot-php)
+* [EDOT PHP-specific configuration options](#options-only-available-in-edot-php)
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)